packages/turbo-types/src/types/config.ts

```diff
@@ -270,3 +270,4 @@ export type OutputMode =
   | "none";
 
 export type AnchoredUnixPath = string;
+export type EnvWildcard = string;
```

Comment:
does this get documented somehow?

Reply:
I've got that over in the wildcard PR; I did this to minimize conflicts. This ends up documented in the schema, but never gets presented to the user in VS Code. 😕